These components use std::this_thread::sleep_for, however sleep_for is not
defined unless #include <thread> has been used.

Signed-off-by: Mark Hatle <mark.hatle@xilinx.com>

diff --git a/tools/Vitis-AI-Runtime/VART/vart/async-runner/src/async_runner.cpp b/tools/Vitis-AI-Runtime/VART/vart/async-runner/src/async_runner.cpp
index 6e48efe48..7c1e380be 100644
--- a/async-runner/src/async_runner.cpp
+++ b/async-runner/src/async_runner.cpp
@@ -21,6 +21,8 @@
 #include <mutex>
 #include <numeric>
 
+#include <thread>
+
 #include "../../runner/src/runner_helper.hpp"
 #include "./batch_tensor_buffer.hpp"
 #include "vitis/ai/collection_helper.hpp"
diff --git a/tools/Vitis-AI-Runtime/VART/vart/util/test/test_thread_pool.cpp b/tools/Vitis-AI-Runtime/VART/vart/util/test/test_thread_pool.cpp
index 85f59685c..0f70fd515 100644
--- a/util/test/test_thread_pool.cpp
+++ b/util/test/test_thread_pool.cpp
@@ -18,6 +18,8 @@
 #include <cstring>
 #include <iostream>
 
+#include <thread>
+
 #include "vitis/ai/env_config.hpp"
 #include "vitis/ai/thread_pool.hpp"
 DEF_ENV_PARAM(NUM_OF_THREADS, "1")
